nimbus.fml.yaml

```diff
@@ -1,8 +1,9 @@
 ---
 channels:
-  - Fennec
-  - FirefoxBeta
-  - Firefox
+  - release
+  - beta
+  - nightly
+  - developer
```

Comment:
What's `nightly` refer to? I ask because our nightly builds are technically beta builds. We only make the distinction with version number being 9000.